uint64 dev_number
uint64 led_number
uint64 led_value
---
bool success
string message
